index.html

```diff
@@ -228,8 +228,9 @@
           <section>
             <h2>Download App Code</h2>
             <p class="left-align">If you haven't already, <br><a href="https://github.com/atroutt/android-beginners/archive/master.zip">download the source code for the first app</a> (zip).</p>
-            <p class="left-align" style="margin-top:40px">Alternatively, clone the repo using git:</p>
-            <pre><code class ="bash">git clone git@github.com:atroutt/android-beginners.git</code></pre>
+            <p class="left-align" style="margin-top:40px">Alternatively, 
+		    the repo using git:</p>
+            <pre><code class ="bash">https://github.com/atroutt/AndroidBeginnerFirstApp.git</code></pre>
```

Comment:
This is the finished app, but I put a copy of the starter app here in the android-beginners repo.

I see you are using the https url instead of ssh, that's probably simpler. In that case, the url should be ```https://github.com/atroutt/android-beginners.git```